packages/MHDDoS/PKGBUILD

```
@@ -0,0 +1,43 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+
+pkgname=MHDDoS
+pkgver=2.4.1
+pkgrel=1
+pkgdesc='Best DDoS Attack Script Python3, (Cyber / DDos) Attack With 56 Methods.'
+arch=('any')
+groups=('blackarch' 'blackarch-dos')
+url='https://github.com/MatrixTM/MHDDoS'
+license=('MIT')
+depends=('python3')
```

this dependency doesn't exist

```suggestion
depends=('python')
```

---

aa sorry i  am new to pkgbuild   i saw python2 ended support so i put 3  .. i will take note